evm/evmone-evm384/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ewasm/bench-build-base:1
+
+LABEL maintainer="Ewasm Team"
+LABEL repo="https://github.com/ewasm/benchmarking"
+LABEL version="1"
+LABEL description="Ewasm benchmarking (evmone-evm384)"
+
+RUN pip3 install durationpy pandas
+
+# install evmone (need CXXFLAGS=-w or the build fails)
```

Comment:
Can you share the build log?

Reply:
```
[100%] Built target benchmark-Release
-- [hunter] Build step successful (dir: /root/.hunter/_Base/bc7da32/22922ba/145276e/Build/benchmark)
-- [hunter] Cache saved: /root/.hunter/_Base/Cache/raw/6da53e85e601304c4a800c161035adf801a514c5.tar.bz2
-- Looking for C++ include filesystem
-- Looking for C++ include filesystem - found
-- [hunter] BENCHMARK_ROOT: /root/.hunter/_Base/bc7da32/22922ba/145276e/Install (ver.: 1.4.0)
-- [hunter] GTEST_ROOT: /root/.hunter/_Base/bc7da32/22922ba/145276e/Install (ver.: 1.8.1)
-- Configuring done
-- Generating done
-- Build files have been written to: /root/evmone-evm384-v1/build
Scanning dependencies of target instructions
Scanning dependencies of target evmone-bench-internal
Scanning dependencies of target loader
Scanning dependencies of target evmone
[  5%] Building CXX object test/internal_benchmarks/CMakeFiles/evmone-bench-internal.dir/find_jumpdest_bench.cpp.o
[  5%] Building C object evmc/lib/loader/CMakeFiles/loader.dir/loader.c.o
[  8%] Building C object evmc/lib/instructions/CMakeFiles/instructions.dir/instruction_metrics.c.o
[ 11%] Building CXX object lib/evmone/CMakeFiles/evmone.dir/analysis.cpp.o
[ 14%] Building C object evmc/lib/instructions/CMakeFiles/instructions.dir/instruction_names.c.o
[ 20%] Linking C static library libevmc-instructions.a
[ 20%] Linking C static library libevmc-loader.a
[ 20%] Built target instructions
Scanning dependencies of target testutils
[ 22%] Building CXX object test/utils/CMakeFiles/testutils.dir/utils.cpp.o
[ 22%] Built target loader
Scanning dependencies of target evmc-vmtester
[ 25%] Building CXX object evmc/tools/vmtester/CMakeFiles/evmc-vmtester.dir/vmtester.cpp.o
[ 28%] Building CXX object lib/evmone/CMakeFiles/evmone.dir/evmone.cpp.o
[ 31%] Building CXX object lib/evmone/CMakeFiles/evmone.dir/execution.cpp.o
[ 34%] Building CXX object evmc/tools/vmtester/CMakeFiles/evmc-vmtester.dir/tests.cpp.o
[ 37%] Building CXX object lib/evmone/CMakeFiles/evmone.dir/instructions.cpp.o
In file included from /root/evmone-evm384-v1/lib/evmone/instructions.cpp:1240:
/root/evmone-evm384-v1/lib/evmone/bigint.h: In function 'void evmone::{anonymous}::square384_64bitlimbs(uint64_t*, const uint64_t*)':
/root/evmone-evm384-v1/lib/evmone/bigint.h:181:17: error: conversion from '__int128 unsigned' to 'uint64_t' {aka 'long unsigned int'} may change value [-Werror=conversion]
  181 |     UINT u = uv >> LIMB_BITS; // / base
/root/evmone-evm384-v1/lib/evmone/bigint.h:189:14: error: conversion from '__int128 unsigned' to 'uint64_t' {aka 'long unsigned int'} may change value [-Werror=conversion]
  189 |       u = uv >> LIMB_BITS; // / base
/root/evmone-evm384-v1/lib/evmone/bigint.h: In function 'void evmone::{anonymous}::montreduce384_64bitlimbs(uint64_t*, uint64_t*, const uint64_t*, uint64_t)':
/root/evmone-evm384-v1/lib/evmone/bigint.h:263:19: error: conversion from '__int128 unsigned' to 'uint64_t' {aka 'long unsigned int'} may change value [-Werror=conversion]
  263 |       carry = sum >> LIMB_BITS; // / b
/root/evmone-evm384-v1/lib/evmone/bigint.h:270:19: error: conversion from '__int128 unsigned' to 'uint64_t' {aka 'long unsigned int'} may change value [-Werror=conversion]
  270 |       carry = sum >> LIMB_BITS; // / b
/root/evmone-evm384-v1/lib/evmone/bigint.h: In function 'void evmone::{anonymous}::montmul384_64bitlimbs(uint64_t*, const uint64_t*, const uint64_t*, const uint64_t*, uint64_t)':
/root/evmone-evm384-v1/lib/evmone/bigint.h:317:18: error: conversion from '__int128 unsigned' to 'uint64_t' {aka 'long unsigned int'} may change value [-Werror=conversion]
  317 |       carry = sum>>LIMB_BITS;
[ 40%] Building CXX object test/internal_benchmarks/CMakeFiles/evmone-bench-internal.dir/memory_allocation.cpp.o
[ 42%] Linking CXX executable ../../bin/evmone-bench-internal
[ 42%] Built target evmone-bench-internal
cc1plus: all warnings being treated as errors
make[2]: *** [lib/evmone/CMakeFiles/evmone.dir/build.make:102: lib/evmone/CMakeFiles/evmone.dir/instructions.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:439: lib/evmone/CMakeFiles/evmone.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 45%] Linking CXX executable ../../bin/evmc-vmtester
[ 48%] Linking CXX static library ../../lib/libtestutils.a
[ 48%] Built target evmc-vmtester
[ 48%] Built target testutils
make: *** [Makefile:163: all] Error 2
The command '/bin/sh -c cd evmone-evm384-v1 && mkdir build &&   cd build && cmake .. -DEVMONE_TESTING=ON && make -j4' returned a non-zero code: 2
```

Reply:
This should not happen because I've fixed these in evmone, but we can figure it out once using the newer evmone release.